cap/config.py

```diff
@@ -106,6 +106,8 @@ def _(x):
 CAP_SEND_MAIL = True
 MAIL_DEFAULT_SENDER = SUPPORT_EMAIL
 
```

Comment:
We can add info about the Mail config for testing purposes here, Something like this.

If you have local mail server setup
```suggestion
# Local Mail Server setup

MAIL_SERVER = 'localhost'
MAIL_PORT = 25
MAIL_USERNAME = ''
MAIL_PASSWORD = ''
MAIL_USE_TLS = False
MAIL_USE_SSL = True
MAIL_SUPPRESS_SEND = False  # Important to add since it is False by default. 

# GMAIL server setup

MAIL_SERVER = 'smtp.gmail.com'
MAIL_PORT = 465
MAIL_USERNAME = '<gmail_id>@gmail.com'
MAIL_PASSWORD = '<password>'  # https://stackoverflow.com/a/73383460/7994074
MAIL_USE_TLS = False
MAIL_USE_SSL = True
MAIL_SUPPRESS_SEND = False # Important to add since it is False by default. 
```